test/index.spec.js

```diff
@@ -1,11 +1,14 @@
-/* eslint-disable no-undef, react/prop-types, react/no-multi-comp, no-unused-expressions */
+/* global chai, expect, sinon */
```

Comment:
Neat, wasn't aware there was a workaround to `no-undef` for stuff like this. Guessing ESLint provides this `global` helper?